pkg/api/playlist.go

```diff
@@ -16,20 +17,20 @@ func (hs *HTTPServer) ValidateOrgPlaylist(c *models.ReqContext) {
 		c.JsonApiErr(http.StatusBadRequest, "id is invalid", nil)
 		return
 	}
-	query := models.GetPlaylistByIdQuery{Id: id}
-	err = hs.SQLStore.GetPlaylist(c.Req.Context(), &query)
```

Comment:
I thought for now we will still use the sqlstore methods in the api, am I wrong?